### PR TITLE
refs #5297 fixes ui bugs in employer portal production

### PR DESCRIPTION
**Pull request by withintheruins14**
### Redmine ticket(s)
https://devops.dchbx.org/redmine/issues/5297

### Local build result

```
3906 examples, 0 failures, 77 pendings

Took 200 seconds (3:20)
48 scenarios (48 passed)
916 steps (916 passed)
7m42.007s
```

### Latest rebase/merge tag
*

---

### Data ticket(s) Followup
* (redmine links here - optional)

### Related Pull Requests
* (github links here - optional)

---

#### Peer Review
* (For code review)

#### Functional Testing
* (For testing locally)

#### Deployment
* (for release manager and/or build manager)

### CODE REVIEW

File: app/assets/stylesheets/custom.scss.erb [hunk 1]
```diff
@@ -4938,7 +4938,3 @@ input:required:valid {
 #new_announcement .form-group {
   margin-bottom: 10px;
 }
-
-.alert-warning {
```
Comment (Kevin-Wei) [inline app/assets/stylesheets/custom.scss.erb]: @hlee this ok?

File: app/views/employers/employer_profiles/my_account/_census_employees.html.erb [hunk 2]
```diff
@@ -45,9 +45,9 @@
...
-<%= link_to 'Add New Employee', new_employers_employer_profile_census_employee_path(@employer_profile, tab: 'employees'), class: "btn btn-default btn-lg" %>
-<%= link_to 'Upload Employee Roster', new_employers_employer_profile_census_employee_path(@employer_profile, modal: 'upload'), remote: 'true', class: "btn btn-default btn-lg" %>
-<%= link_to 'Upload Employee Roster', employers_employer_profile_export_census_employees_path(@employer_profile, format: :csv), class: "btn btn-default btn-lg" %>
+<%= link_to "<i class='fa fa-user-plus' aria-hidden='true'></i> Add New Employee".html_safe, new_employers_employer_profile_census_employee_path(@employer_profile, tab: 'employees'), class: "btn btn-default" %>
+<%= link_to "<i class='fa fa-file-text-o' aria-hidden='true'></i> Upload Employee Roster".html_safe, new_employers_employer_profile_census_employee_path(@employer_profile, modal: 'upload'), remote: 'true', class: "btn btn-default" %>
+<%= link_to "<i class='fa fa-cloud-download' aria-hidden='true'></i> Upload Employee Roster".html_safe, employers_employer_profile_export_census_employees_path(@employer_profile, format: :csv), class: "btn btn-default" %>
```
Comment (Kevin-Wei) [inline app/views/employers/employer_profiles/my_account/_census_employees.html.erb]: This still says upload twice... @withintheruins14 ?

File: app/assets/stylesheets/custom.scss.erb (hunk 1 above)
Comment (withintheruins14) [inline app/assets/stylesheets/custom.scss.erb]: yes haha

File: app/views/employers/employer_profiles/my_account/_census_employees.html.erb (hunk 2 above)
Comment (withintheruins14) [inline app/views/employers/employer_profiles/my_account/_census_employees.html.erb]: not okay, thanks

File: app/views/employers/employer_profiles/my_account/_census_employees.html.erb [hunk 3]
```diff
@@ -45,9 +45,9 @@
...
-<%= link_to 'Add New Employee', new_employers_employer_profile_census_employee_path(@employer_profile, tab: 'employees'), class: "btn btn-default btn-lg" %>
-<%= link_to 'Upload Employee Roster', new_employers_employer_profile_census_employee_path(@employer_profile, modal: 'upload'), remote: 'true', class: "btn btn-default btn-lg" %>
-<%= link_to 'Upload Employee Roster', employers_employer_profile_export_census_employees_path(@employer_profile, format: :csv), class: "btn btn-default btn-lg" %>
+<%= link_to "<i class='fa fa-user-plus' aria-hidden='true'></i> Add New Employee".html_safe, new_employers_employer_profile_census_employee_path(@employer_profile, tab: 'employees'), class: "btn btn-default" %>
+<%= link_to "<i class='fa fa-file-text-o' aria-hidden='true'></i> Upload Employee Roster".html_safe, new_employers_employer_profile_census_employee_path(@employer_profile, modal: 'upload'), remote: 'true', class: "btn btn-default" %>
+<%= link_to "<i class='fa fa-cloud-download' aria-hidden='true'></i> Download Employee Roster".html_safe, employers_employer_profile_export_census_employees_path(@employer_profile, format: :csv), class: "btn btn-default" %>
```
Comment (Kevin-Wei) [inline app/views/employers/employer_profiles/my_account/_census_employees.html.erb]: baller 👍

**Pull request merged**